recipes/sphinx-changelog/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "sphinx-changelog" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a63025a9360a40ebcd9ba1e156f10063a80b4c571761838659165d5aa68094c2
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools-scm
+    - pip
+  run:
+    - python >=3.6
+    - sphinx
+    - towncrier >=22.8.0
+
+test:
+  imports:
+    - sphinx_changelog
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://openastronomy.org
+  summary: A changelog renderer for sphinx.
+  description: |
+    This package provides a sphinx extension designed to facilitate rendering
+    changelogs into your sphinx documentation.
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file:
+    - licenses/LICENSE.rst
+  doc_url: https://sphinx-changelog.readthedocs.io/en/latest/
+  dev_url: https://github.com/OpenAstronomy/sphinx-changelog
+
+extra:
+  recipe-maintainers:
+    - BSGalvan
```

```suggestion
    - BSGalvan
    - Cadair
```

I am happy to help maintain this (as the primary author of the package).